universum/modules/vcs/github_app_vcs.py

```diff
@@ -220,12 +220,13 @@ def code_report_to_review(self, report):
         comments = []
         for path, issues in report.items():
             if path in commit_files:
-                for issue in issues:
-                    comments.append(dict(path=path,
-                                         message=issue['message'],
-                                         start_line=issue['line'],
-                                         end_line=issue['line'],
-                                         annotation_level="warning"))
+                break
```

Comment:
I think the break here is incorrect. It stops processing of report as soon as issue in file on review is reached.